tests/unit/utils/test_errors_parsing.py

```diff
@@ -6,46 +6,49 @@
 
 from ols.utils import errors_parsing
 
+BAD_REQUEST_STATUS_CODE = 400
+
 
 def test_parse_generic_llm_error_on_bad_request_error_without_info():
     """Test the parse_generic_llm_error function when BadRequestError is passed."""
-    status_code = 400
-    response = Response(status_code=status_code, request={})
+    expected_status_code = BAD_REQUEST_STATUS_CODE
+    response = Response(status_code=expected_status_code, request={})
     error = BadRequestError("Exception", response=response, body=None)
 
     # try to parse the exception
     status_code, error_message, cause = errors_parsing.parse_generic_llm_error(error)
 
     # check the parsed error
-    assert status_code == status_code
+    assert status_code == expected_status_code
     assert error_message == "Exception"
     assert cause == "Exception"
 
 
 def test_parse_generic_llm_error_on_bad_request_error_with_info():
     """Test the parse_generic_llm_error function when BadRequestError is passed."""
-    status_code = 400
+    expected_status_code = BAD_REQUEST_STATUS_CODE
     message = "Exception message"
-    response = Response(status_code=status_code, request={})
+    response = Response(status_code=expected_status_code, request={})
     error = BadRequestError("Exception", response=response, body={"message": message})
 
     # try to parse the exception
     status_code, error_message, cause = errors_parsing.parse_generic_llm_error(error)
 
     # check the parsed error
-    assert status_code == status_code
+    assert status_code == expected_status_code
     assert error_message == message
     assert cause == "Exception"
 
 
 def test_parse_generic_llm_error_on_api_response_exception_without_info():
     """Test the parse_generic_llm_error function when ApiResponseException is passed."""
+    expected_status_code = BAD_REQUEST_STATUS_CODE
     error = ApiResponseException(
         response={
             "error": "",
             "message": "",
             "extensions": {},
-            "status_code": 400,
+            "status_code": BAD_REQUEST_STATUS_CODE,
```

Comment:
If we're assigning `expected_status_code = BAD_REQUEST_STATUS_CODE`, I imagine it's because within the test, we'll want to use expected_status_code instead of BAD_REQUEST_STATUS_CODE